devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusAddExtensions.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.gradle.tasks;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+import io.quarkus.cli.commands.AddExtensions;
+import io.quarkus.cli.commands.writer.FileProjectWriter;
+import io.quarkus.generators.BuildTool;
+
+/**
+ * @author <a href="mailto:stalep@gmail.com">Ståle Pedersen</a>
+ */
+public class QuarkusAddExtensions extends QuarkusTask {
+
+    public QuarkusAddExtensions() {
+        super("Adds some available quarkus extensions to the current project");
+    }
+
+    private List<String> extensionsToAdd;
+
+    @Option(option = "extensions-to-add", description = "Configures the extensions to be added.")
```

Comment:
Could we have a shorter name? E.g. simply `extensions`?

Reply:
Yes, I rename the field because getExtensions already exist, but we can keep the simple option name ;o)